quickstart/101-key-vault-key/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "proxy_key" {
+  value = azurerm_key_vault_key.key
```

Comment:
Do we want to export the entire key here? Might make more sense to only output the key id or versionless_id.